content/manuals/engine/storage/drivers/overlayfs-driver.md

```diff
@@ -12,13 +12,17 @@ This page refers to the Linux kernel driver as `OverlayFS` and to the Docker
 storage driver as `overlay2`.
 
 > [!NOTE]
->
+> Docker Engine 29.0 and later uses the
+> [containerd image store](/manuals/engine/storage/containerd.md) by default.
+> The `overlay2` driver is a legacy storage driver. For more information, see
```

Comment:
Maybe add something like "it has been superseded by `overlayfs` containerd snapshotter"?